tests/org.pitest.pitclipse.ui.tests/src/org/pitest/pitclipse/ui/behaviours/pageobjects/Views.java

```diff
@@ -1,118 +0,0 @@
-/*******************************************************************************
```

Comment:
This class now exists as `Console.java`, which imo is more fitting and we can get the text from the console there. This would allow us to check the console for some content in the future.

Reply:
I indeed missed that. @JKutscha it looks like some methods have been lost though (i.e., `clear`, `close` and `show`). Even if they're not used anymore they look like useful methods that we'll likely use in the future, and since they're already implemented we may want to move them to the `Console` class...

I don't have a strong opinion on that but I I'd prefer the methods to be kept because, as I said, they are very likely to be used in the future.

Reply:
Will do.

Reply:
Opps... sorry I have already merged... I'll wait for the next PR